metadata/src/main/java/com/redhat/lightblue/metadata/ValueGenerator.java

```diff
@@ -0,0 +1,42 @@
+package com.redhat.lightblue.metadata;
+
+import java.io.Serializable;
+import java.util.Properties;
+
+/**
+ * Represents value generator definition located in entityInfo.
+ *
+ * @author mpatercz
+ *
+ */
+public class ValueGenerator implements Serializable {
+
+    private static final long serialVersionUID = 1L;
+
+    public enum ValueGeneratorType {
+        IntSequance;
+    }
+
```

Comment:
The enum I created (ValueGeneratorType.IntSequence) defines not only the type of generated value (int), but also how it is generated (sequence). Are you proposing to define the type of the generated value using com.redhat.lightblue.metadata.types.Type and having a separate generator type (sequance, random, etc.) as enum?

You never said you want a new value type, it's just what I assumed thinking about generator type. Perhaps this is the source of the confusion.